traffic_ops/client/server_server_capabilities.go

```diff
@@ -0,0 +1,79 @@
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/url"
+	"strconv"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+)
+
+const (
+	API_v14_Server_Server_Capabilities = "/api/1.4/server_server_capabilities"
```

Comment:
For this const we should probably use `APIServerServerCapabilities = apiBase + "/server_server_capabilities"` so that the Go client will make requests at the latest API version whenever `apiBase` is updated to point to the latest API version.